PocketKit/Sources/PocketKit/PocketAppDelegate.swift

```diff
@@ -80,6 +80,23 @@ public class PocketAppDelegate: UIResponder, UIApplicationDelegate {
         Textiles.initialize()
         refreshCoordinator.initialize()
 
+        do {
+            let migration = LegacyUserMigration(
+                userDefaults: userDefaults,
+                encryptedStore: PocketEncryptedStore(),
+                appSession: appSession
+            )
+
+            let attempted = try migration.perform()
+            if attempted {
+                Crashlogger.breadcrumb(category: "launch", level: .info, message: "Legacy user migration required; running.")
+            } else {
+                Crashlogger.breadcrumb(category: "launch", level: .info, message: "Legacy user migration not required; skipped.")
+            }
+        } catch {
+            Crashlogger.capture(error: error)
+        }
```

Comment:
should we consider putting this piece in `SharedPocketKit` since we are using it for both the app and extension?